geti_sdk/data_models/configuration.py

```diff
@@ -207,7 +207,7 @@ def get_component_configuration(
         )
 
 
-@attr.s(auto_attribs=True)
+@attr.define(auto_attribs=True)
```

Comment:
Shouldn't this also have `slots=False`, then? Since it inherits from `Configuration`?

Reply:
I am not too sure but it makes sense for slot classes to inherit from slot classes and vice versa.